com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/Messages.java

```diff
@@ -18,6 +18,10 @@
 
     // ERROR log messages
     public static final String MTA_NOT_FOUND = "MTA with id \"{0}\" does not exist";
+    public static final String MTAS_NOT_FOUND_BY_NAME = "MTAs with name \"{0}\" not found";

```

Comment:
I think it would be more consistent if the message said `MTAs with name \"{0}\" do not exist".